code/modules/surgery/bodyparts/bodyparts.dm

```diff
@@ -71,6 +71,8 @@
 	var/medium_burn_msg = "blistered"
 	var/heavy_burn_msg = "peeling away"
 
+	var/render_like_organic = FALSE // forces limb to render as if it were an organic limb
```

Comment:
I want to point out that `render_like_organic` is some serious shitcode that you don't want. It snowflakes robotic limbs in the limb code. 

I would assume that citadel has better solutions to this, what with how many different bodytypes/sprites/etc are in play on bodyparts, but I might be wrong on this.

Reply:
I am not currently aware of a better solution because of how limbs are handled by the code, but I would be happy to be informed of any better ways which I could try implement.